templates/core/.env.sample

```diff
@@ -15,3 +15,7 @@ SWAGGER_UI_CLIENT_ID=__CHANGE_ME__
 # Automation accounts (e.g. bundle registration)
 AUTOMATION_ADMIN_ACCOUNT_CLIENT_ID=__CHANGE_ME__
 AUTOMATION_ADMIN_ACCOUNT_CLIENT_SECRET=__CHANGE_ME__
+
+# Bundle Registration via Automation needs to know TRE_URL.
```

Comment:
Thinking....

Can we put these "optional" or rather CI/CD type variables under a separate header saying "Automated deployments such as deployments which are part of CI/CD pipelines will need the following variables:"

For users wanting to kick the tires, they do not need to set these, or the above automation env vars, so is slightly confusing.

Happy to discuss.

Reply:
so, comment all three of these out in the example?

Reply:
Great. Thanks.